[2024-07-13T14:25:29.464+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T14:25:29.849+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-07-13T14:25:29.917+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-07-13T14:25:29.919+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-13T14:25:30.148+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): extract.extract_orders> on 2024-06-27 00:00:00+00:00
[2024-07-13T14:25:30.196+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3777) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-13T14:25:30.206+0000] {standard_task_runner.py:63} INFO - Started process 3821 to run task
[2024-07-13T14:25:30.218+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'extract.extract_orders', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpffs9nffs']
[2024-07-13T14:25:30.360+0000] {standard_task_runner.py:91} INFO - Job 146: Subtask extract.extract_orders
[2024-07-13T14:25:30.594+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-13T14:25:31.419+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-13T14:25:33.067+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='extract.extract_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-07-13T14:25:33.082+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T14:25:33.196+0000] {base.py:84} INFO - Using connection ID 'post_conn' for task execution.
[2024-07-13T14:25:33.484+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T14:25:33.488+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/elt.py", line 49, in extract
    return extract_func(
           ^^^^^^^^^^^^^
  File "/opt/airflow/dags/el_module/extractors.py", line 11, in extract_func
    ).get_conn()
      ^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py", line 175, in get_conn
    self.conn = psycopg2.connect(**conn_args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: connection to server at "ep-bold-forest-89569418.ap-southeast-1.aws.neon.tech" (13.228.184.177), port 5432 failed: ERROR:  password authentication failed for user 'dataninja'
connection to server at "ep-bold-forest-89569418.ap-southeast-1.aws.neon.tech" (13.228.184.177), port 5432 failed: ERROR:  connection is insecure (try using `sslmode=require`)

[2024-07-13T14:25:33.639+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=ELT-PROJECT, task_id=extract.extract_orders, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240713T142529, end_date=20240713T142533
[2024-07-13T14:25:33.711+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 146 for task extract.extract_orders (connection to server at "ep-bold-forest-89569418.ap-southeast-1.aws.neon.tech" (13.228.184.177), port 5432 failed: ERROR:  password authentication failed for user 'dataninja'
connection to server at "ep-bold-forest-89569418.ap-southeast-1.aws.neon.tech" (13.228.184.177), port 5432 failed: ERROR:  connection is insecure (try using `sslmode=require`)
; 3821)
[2024-07-13T14:25:33.763+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-07-13T14:25:33.829+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-14T03:58:09.276+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T03:58:09.363+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-07-14T03:58:09.390+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-07-14T03:58:09.391+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-14T03:58:09.469+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): extract.extract_orders> on 2024-06-27 00:00:00+00:00
[2024-07-14T03:58:09.519+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'extract.extract_orders', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '671', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpc0p0hsn0']
[2024-07-14T03:58:09.534+0000] {standard_task_runner.py:91} INFO - Job 671: Subtask extract.extract_orders
[2024-07-14T03:58:09.526+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9475) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T03:58:09.541+0000] {standard_task_runner.py:63} INFO - Started process 9506 to run task
[2024-07-14T03:58:09.631+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-14T03:58:09.892+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.extract.extract_orders scheduled__2024-06-27T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-14T03:58:10.585+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='extract.extract_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-07-14T03:58:10.594+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T03:58:10.694+0000] {base.py:84} INFO - Using connection ID 'post_conn' for task execution.
[2024-07-14T03:58:11.521+0000] {python.py:237} INFO - Done. Returned value was:      orderID customerID  employeeID  ... shipRegion shipPostalCode  shipCountry
0      10663      BONAP           2  ...        NaN          13008       France
1      10664      FURIB           1  ...        NaN           1675     Portugal
2      10665      LONEP           1  ...         OR          97219          USA
3      10666      RICSU           7  ...        NaN           1204  Switzerland
4      10667      ERNSH           7  ...        NaN           8010      Austria
..       ...        ...         ...  ...        ...            ...          ...
825    10658      QUICK           4  ...        NaN          01307      Germany
826    10659      QUEEN           7  ...         SP      05487-020       Brazil
827    10660      HUNGC           8  ...         OR          97827          USA
828    10661      HUNGO           7  ...   Co. Cork            NaN      Ireland
829    10662      LONEP           3  ...         OR          97219          USA

[830 rows x 14 columns]
[2024-07-14T03:58:11.782+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T03:58:13.102+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=extract.extract_orders, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240714T035809, end_date=20240714T035813
[2024-07-14T03:58:14.024+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T03:58:14.726+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-14T03:58:14.744+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
